source/MaterialXTest/ShaderValid.cpp

```diff
@@ -402,95 +397,7 @@ TEST_CASE("GLSL geometry", "[shadervalid]")
-    geomcolor2->setParameterValue("index", 1, "integer");
-    output->setName("geomcolor2_output");
-    output->setType("color3");
-    output->setConnectedNode(geomcolor2);
-    runValidation(".", "geomcolor2_attributes", output, validator, shaderGenerator, orthographicView, doc, log, true);
-    nodeGraph->removeNode(geomcolor2->getName());
-
-    // Tangent stream test.
-    nodeGraph->setName("tangent_nodegraph");
-    mx::NodePtr tangent1 = nodeGraph->addNode("tangent", "tangent1", "vector3");
-    tangent1->setParameterValue("index", 0, "integer");
-    output->setName("tangent_output");
-    output->setType("vector3");
-    output->setConnectedNode(tangent1);
-    runValidation(".", "tangent_attributes", output, validator, shaderGenerator, orthographicView, doc, log, true);
-    nodeGraph->removeNode(tangent1->getName());
-
-    // Bitangent stream test
-    nodeGraph->setName("bitangent_nodegraph");
-    mx::NodePtr bitangent1 = nodeGraph->addNode("bitangent", "bitangent1", "vector3");
-    tangent1->setParameterValue("index", 0, "integer");
-    output->setName("bitangent_output");
-    output->setType("vector3");
-    output->setConnectedNode(bitangent1);
-    runValidation(".", "bitangent_attributes", output, validator, shaderGenerator, orthographicView, doc, log, true);
-    nodeGraph->removeNode(bitangent1->getName());
-
-    // UV stream test
-    nodeGraph->setName("texcoord1_nodegraph");
-    mx::NodePtr texcoord1 = nodeGraph->addNode("texcoord", "texcoord1", "vector2");
-    texcoord1->setParameterValue("index", 0, "integer");
-    mx::NodePtr swizzle1 = nodeGraph->addNode("swizzle", "uv_set0", "vector3");
-    swizzle1->setConnectedNode("in", texcoord1);
-    swizzle1->setParameterValue("channels", std::string("xy0"));
-    output->setName("texcoord1_output");
-    output->setType("vector3");
-    output->setConnectedNode(swizzle1);
-    runValidation(".", "texcoord1_attributes", output, validator, shaderGenerator, orthographicView, doc, log, true);
-    nodeGraph->removeNode(texcoord1->getName());
-    nodeGraph->removeNode(swizzle1->getName());
-
-    // Second UV stream test
-    nodeGraph->setName("texcoord2_nodegraph");
-    mx::NodePtr texcoord2 = nodeGraph->addNode("texcoord", "texcoord2", "vector2");
-    texcoord2->setParameterValue("index", 1, "integer");
-    mx::NodePtr swizzle2 = nodeGraph->addNode("swizzle", "uv_set1", "vector3");
-    swizzle2->setConnectedNode("in", texcoord2);
-    swizzle2->setParameterValue("channels", std::string("xy0"));
-    output->setName("texcoord2_output");
-    output->setType("vector3");
-    output->setConnectedNode(swizzle2);
-    runValidation(".", "texcoord2_attributes", output, validator, shaderGenerator, orthographicView, doc, log, true);
-    nodeGraph->removeNode(texcoord2->getName());
-    nodeGraph->removeNode(swizzle2->getName());
-
-    // Image test
+    // Image test - TODO make this driven by input data file
```

Comment:
Geom stream test in data file now. When we add the image data file we can delete this.